test-workspace/src/main/scala/example/User.scala

```diff
@@ -5,6 +5,6 @@ import java.nio.file.Paths
 case class User(name: String, age: Int)
 
 object User {
-  val sum: Int = 432
+  val sum: Int = ""
```

Comment:
Accidental change?

Reply:
Yes, was experimenting. Forgot to undo the diff, but it doesn't matter much I suppose.

Reply:
Removed.